index.html

```diff
@@ -0,0 +1,153 @@
+<!doctype html>
+<html lang="en">
+<head>
+  <meta charset="utf-8">
+  <title>Xplosion</title>
+  <meta name="description" content="Xplosion site">
+  <meta name="author" content="Michal Mrugala">
+  <meta content="width=device-width, initial-scale=1, maximum-scale=1, user-scalable=0" name="viewport">
+  <!-- styles -->
+  <link rel="stylesheet" href="/css/style.css">
+</head>
+<body>
+  <div class="wrapper">
+    <header class="main-header clearfix">
```

Comment:
Please put roles within the document.

![img](https://patrickhlauke.github.io/aria/presentation/pictures/blog-structure-after-aria.png)